### PR TITLE
Server side rendering

### PR DESCRIPTION
**Pull request by redharp**
@EvaFro  please review my server-side rendering code, thanks 😺

### CODE REVIEW

File: proxy/templates/scripts.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+module.exports = (items) => `
+  <script src="/lib/react.development.js"></script>
```
Comment (EvaFro) [inline proxy/templates/scripts.js]: I am guessing that these are local copies of the .js library files needed? So they arn't worth loading and looking at right?

**Review by EvaFro — APPROVED** [review]
@Frogz1  I just have one question on your libraries. But other then that this looks great!

File: proxy/templates/scripts.js (hunk 1 above)
Comment (redharp) [reply proxy/templates/scripts.js]: Yes exactly, I replicated what Fred did and load the react files from disk instead of an http call

Comment (EvaFro) [reply proxy/templates/scripts.js]: Cool! I just wanted to make sure.

**Pull request merged**